templates/csharp/api-plugin-from-scratch-bearer/.{{NewProjectTypeName}}/GettingStarted.md.tpl

```diff
@@ -0,0 +1,29 @@
+# Welcome to Teams Toolkit!
+
+## Quick Start
+
+> **Prerequisites**
+>
+> To run this app template in your local dev machine, you will need:
+>
+> - [Visual Studio 2022](https://aka.ms/vs) 17.9 or higher and [install Teams Toolkit](https://aka.ms/install-teams-toolkit-vs)
+> - A [Microsoft 365 account for development](https://docs.microsoft.com/microsoftteams/platform/toolkit/accounts)
+
+1. In the debug dropdown menu, select Dev Tunnels > Create a Tunnel (set authentication type to Public) or select an existing public dev tunnel
+</br>![image](https://raw.githubusercontent.com/OfficeDev/TeamsFx/dev/docs/images/visualstudio/debug/create-devtunnel-button.png)
+2. Right-click the '{{NewProjectTypeName}}' project and select Teams Toolkit > Prepare Teams App Dependencies
+3. If prompted, sign in with a Microsoft 365 account for the Teams organization you want to install the app to.
+4. Press F5, or select the `Debug > Start Debugging` menu in Visual Studio to start your app
+</br>![image](https://raw.githubusercontent.com/OfficeDev/TeamsFx/dev/docs/images/visualstudio/debug/debug-button.png)
+5. When Teams launches in the browser, you can open the Copilot app and send a prompt to trigger your plugin.
+6. Send a message to Copilot to find an NuGet package information. For example: Find the NuGet package info on Microsoft.CSharp.
```

Comment:
This step description needs to be adjusted, since this template is not for finding an NuGet package.

Reply:
I'll synchronize with the OAuth template, but I'm not sure if the readme file for the no-auth template has the same problem。